sentry-android/build.gradle.kts

```diff
@@ -5,6 +5,17 @@ plugins {
     kotlin("android")
     id(Config.Deploy.novodaBintray)
     id(Config.QualityPlugins.gradleVersions)
+    distribution
+}
+
+// overwrite distributions config from the root project
+distributions {
```

Comment:
should this go to all the other "android modules"?

Reply:
Would it work to add all of these paths to the root?
Does it fail if the directory doesn't exist or something?

Reply:
Good idea @bruno-garcia, it works.